### PR TITLE
feat: add skeleton pages for initial Storage service docs

### PR DESCRIPTION
**Pull request by cprice404**

### CODE REVIEW

File: docs/storage/develop/index.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+Here is an example of how to construct a `StorageClient`:
+
+<SdkExampleTabs snippetId={'API_Storage_InstantiateClient'} />
+
+## Instantiating credential providers using Momento API keys
+
+You need to provide a Momento API key when instantiating a Momento client. If you don't have one yet, you can get one from the [Momento Web Console](https://console.gomomento.com/). Once you have a token, provide it to Momento SDKs when you create an instance of `CredentialProvider`. There are convenient factory methods provided to construct a `CredentialProvider` object, either from an environment variable or from a String. Below is an example of how to instantiate `CredentialProvider` from an environment variable:
+
+<SdkExampleTabs snippetId={'API_CredentialProviderFromEnvVar'} />
```
Comment (anitarua) [inline docs/storage/develop/index.md:29]: looks like the JS snippets are still using the old style of passing in objects: https://github.com/momentohq/client-sdk-javascript/blob/main/examples/nodejs/cache/doc-example-files/doc-examples-js-apis.ts#L113-L120

File: docs/storage/develop/api-reference/index.mdx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+sidebar_label: API Reference
+description: Learn the API calls you need to know and how to use them with Momento Storage.
+---
+
+import { SdkExampleTabs } from "@site/src/components/SdkExampleTabs";
+// This import is necessary even though it looks like it's un-used; The inject-example-code-snippet
+// plugin will transform instances of SdkExampleTabs to SdkExampleTabsImpl
+import { SdkExampleTabsImpl } from "@site/src/components/SdkExampleTabsImpl";
+
+# API reference for Momento Storage
```
Comment (anitarua) [inline docs/storage/develop/api-reference/index.mdx:14]: doesn't need to happen right away, but should we start making api reference docs in the more stripe-ish style as used for [leaderboards](https://docs.momentohq.com/leaderboards/api-reference)?

**Review by anitarua — DISMISSED** [review]
overall lgtm!

File: docs/sdks/nodejs/storage.mdx [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+## Write an item to a store
+A simple example of doing a put operation.
+
+<SdkExampleCodeBlock language={'javascript'} snippetId={'API_Storage_Put'} />
+
+## Read an item from a store
+This is an example of a simple read operation to get an item from a sore.
+
+<SdkExampleCodeBlock language={'javascript'} snippetId={'API_Storage_Put'} />
```
Comment (danielamiao) [inline docs/sdks/nodejs/storage.mdx]: it says Put even though you probably want to link to Get?

File: docs/storage/develop/index.md (hunk 1 above)
Comment (cprice404) [reply docs/storage/develop/index.md:29]: ty, fixed in https://github.com/momentohq/client-sdk-javascript/pull/1388 but I think it's fine to let that trickle in to the docs whenever the build timing works out.

File: docs/storage/develop/api-reference/index.mdx (hunk 2 above)
Comment (cprice404) [reply docs/storage/develop/api-reference/index.mdx:14]: i completely forgot about this. I think I'd rather we do cache/topics/storage all at the same time for that, but i agree with you. maybe you can help me remember this in our next sprint planning.

File: docs/sdks/nodejs/storage.mdx (hunk 3 above)
Comment (cprice404) [reply docs/sdks/nodejs/storage.mdx]: thank you!

Comment (cprice404) [inline docs/sdks/nodejs/storage.mdx]:
```suggestion
<SdkExampleCodeBlock language={'javascript'} snippetId={'API_Storage_Get'} />
```

Comment (cprice404) [reply docs/sdks/nodejs/storage.mdx]: fixed in https://github.com/momentohq/public-dev-docs/pull/1048/commits/3db326cf82cc07d18a23c34eb677316aeb3d119b

**Review by danielamiao — APPROVED** [review]

**Pull request merged**